src/GraphQL/Types/ISchema.cs

```diff
@@ -110,6 +110,19 @@ public interface ISchema : IProvideMetadata, IProvideDescription
         /// </summary>
         void RegisterType(Type type);
 
+        /// <summary>
+        /// Registers type mapping from CLR type to GraphType.
+        /// <br/>
+        /// These mappings are used for type inference when constructing fields using expressions:
+        /// <br/>
+        /// <c>
+        /// Field(x => x.Filters);
+        /// </c>
+        /// </summary>
+        /// <param name="clrType">The CLR property type from which to infer the GraphType.</param>
+        /// <param name="graphType">Inferred GraphType.</param>
+        void RegisterTypeMapping(Type clrType, Type graphType);
```

Comment:
I see we have removed all the `GraphTypeTypeRegistry` methods and replaced it with a single method here on `ISchema`, but no way to enumerate those registrations.  I think we need `AllTypeRegistrations` or something also.  My code is extremely reliant on the `GraphTypeTypeRegistry`, and I know I will have some code that will need to do something like this:

```
if not already registered clr type x, then register clr type x with graph type y
```

Reply:
See `EnumerateTypeMappings`.